keyboards/rad/keymaps/via/vial.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This appears to be a VIAL json. 
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

> Do not include VIAL json files in the PR. These do not belong in the QMK repository as they are not used by QMK firmware – they belong in the VIAL